orca-applications/src/main/groovy/com/netflix/spinnaker/orca/applications/tasks/DeleteApplicationTask.groovy

```diff
@@ -42,15 +47,17 @@ class DeleteApplicationTask extends AbstractFront50Task {
           if (re.response?.status == 404) {
             return TaskResult.SUCCEEDED
           }
-          log.error("Could not create or update application permission", re)
+          log.error("Could not delete application permission: {}", re.toString(), re)
```

Comment:
I thought the logger already dumps out a string representation of the exception (in addition to the stack trace) when you pass the exception as the last argument.

Reply:
Yeah, looking over the docs and some examples again, I think you're right. I remember this talk/chat with @dreynaud a long time ago where he discussed best practices around exception logging, and was under the impression that was not the case, but I'm probably misremembering. Will undo.